app/star-rating.sass

```diff
@@ -9,4 +9,32 @@ h1
    margin: 0
 
 #app
-   font-size: 3em
\ No newline at end of file
+   font-size: 3em
+
+.star-rating
+    display: inline-block
+    unicode-bidi: bidi-override
+    direction: rtl
+    border-width: 0px
+
+    .star-rating__clicked::before,
+    .star-rating__clicked ~ .star-rating__label:before
+        font-family: "Font Awesome 5 Free"
+        font-weight: 900
+        content: "\f005"
```

Comment:
Can you eliminate duplication of `content` declaration?

Reply:
see commit  "reformates css file structure and removes duplicate code for displaying icon on hover""